SYS_ATL/new_eff.py

```diff
@@ -510,7 +519,8 @@ def _lsstr(ls, prec=0):
         return "∅"
     elif isinstance(ls, LS.Point):
         coords = ','.join([str(a) for a in ls.coords])
-        return f"{{({ls.name},{coords})}}"
+        typ    = f":{ls.type}" if ls.type else ""
+        return f"{{({ls.name},{coords}{typ})}}"
```

Comment:
On line 550 below, `return f"alloc({ls.name},{arg})"` references `arg` which does not exist.

Reply:
Thanks; just caught this yesterday